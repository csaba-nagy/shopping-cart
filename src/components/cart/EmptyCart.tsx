import React from 'react'
import { Link } from 'react-router-dom'

function EmptyCart() {
  return (
    <div>
      <h3>Your cart is empty!</h3>
      <p>Please check our <Link to='/products'>products</Link></p>
    </div>
  )
}

export default EmptyCart
