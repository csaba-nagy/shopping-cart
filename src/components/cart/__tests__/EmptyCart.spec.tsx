import EmptyCart from '../EmptyCart'
import { renderWithProviders } from '../../../test/test-utils'

describe('EmptyCart', () => {
  it('should match with the snapshot', () => {
    const { container, getByText } = renderWithProviders(<EmptyCart />)

    expect(container).toMatchSnapshot()
    expect(getByText(/empty/i)).toBeInTheDocument()
  })
})
