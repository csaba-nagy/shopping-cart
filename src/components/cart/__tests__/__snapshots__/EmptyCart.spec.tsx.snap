// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`EmptyCart > should match with the snapshot 1`] = `
<div>
  <header>
    <a
      class="logo"
      href="/"
    >
      Logo
    </a>
    <nav
      class="navbar"
    >
      <a
        class="navbar-link"
        href="/products"
      >
        Products
      </a>
      <a
        class="navbar-link"
        href="/cart"
      >
        Cart(
        0
        )
      </a>
    </nav>
  </header>
  <div>
    <h3>
      Your cart is empty!
    </h3>
    <p>
      Please check our 
      <a
        href="/products"
      >
        products
      </a>
    </p>
  </div>
  <footer>
    <p>
      Total: 
      0.00
       $
    </p>
  </footer>
</div>
`;
